pkg/core/blockchain.go

```diff
@@ -1674,7 +1674,11 @@ func (bc *Blockchain) initVerificationVM(ic *interop.Context, hash util.Uint160,
 			return ErrInvalidVerificationContract
 		}
 		initMD := cs.Manifest.ABI.GetMethod(manifest.MethodInit)
-		v.LoadScriptWithHash(cs.Script, hash, smartcontract.ReadStates)
+		flag := smartcontract.ReadStates
+		if cs.ID < 0 {
+			flag |= smartcontract.AllowCall
```

Comment:
I think it shouldn't be just for for native contracts. Other contracts may also need to call some additional contract.